diff --git a/security/selinux/hooks.c b/security/selinux/hooks.c
index 36af76e8..7109a2d0 100755
--- a/security/selinux/hooks.c
+++ b/security/selinux/hooks.c
@@ -2565,6 +2565,11 @@ static u32 ptrace_parent_sid(struct task_struct *task)
 	return sid;
 }
 
+#ifdef CONFIG_KSU
+extern bool is_ksu_transition(const struct task_security_struct *old_tsec,
+                              const struct task_security_struct *new_tsec);
+#endif
+
 static int check_nnp_nosuid(const struct linux_binprm *bprm,
 			    const struct task_security_struct *old_tsec,
 			    const struct task_security_struct *new_tsec)
